### PR TITLE
RSDK-7633: Remove extra from response in GetPosition from SLAM

### PR DESCRIPTION
**Pull request by martha-johnston**

### CODE REVIEW

File: services/slam/fake/data_loader.go
Comment (oliviamiller) [inline services/slam/fake/data_loader.go:38]: do we still need the Extra here?

Comment (martha-johnston) [reply services/slam/fake/data_loader.go:38]: yeah I wasn't sure about this but it's used in the `fakePosition` function. was hoping @JohnN193 has some insight on how necessary it is

Comment (JohnN193) [reply services/slam/fake/data_loader.go:38]: i don't remember if the positions in artifact use the extra field. I believe they do not but I would have to double check.

they should be in `slam/example_cartographer_outputs/viam-office-02-22-3/position ` or something like that. I'll be able to look myself later today

Comment (martha-johnston) [reply services/slam/fake/data_loader.go:38]: I'm not sure where this file is

Comment (JohnN193) [reply services/slam/fake/data_loader.go:38]: Screenshot 2024-05-22 at 11 21 18 AM
no worries, basically we use artifact to store any files we need for testing & for fake components/services. More info can be found [here](https://viam.atlassian.net/wiki/spaces/ENG/pages/262371/Viam+Engineering+Guidelines#Testing-with-big-data)

it does look like the fake datasets still use the extras field, so we should not remove that from this pr. Might be worth uploading a new dataset someday but I do not believe thats necessary for this pr.

**Review by JohnN193 — APPROVED** [review]

File: services/slam/fake/data_loader.go
Comment (martha-johnston) [reply services/slam/fake/data_loader.go:38]: okay sounds good I'll leave it

**Comment by martha-johnston** [comment]
@oliviamiller i'll wait til you're ready to merge api just in case there's any issues, although I don't think there will be.

**Comment by oliviamiller** [comment]
> @oliviamiller i'll wait til you're ready to merge api just in case there's any issues, although I don't think there will be.

are there other SDK changes needed or is this the only one?

**Comment by martha-johnston** [comment]
> > @oliviamiller i'll wait til you're ready to merge api just in case there's any issues, although I don't think there will be.
> 
> are there other SDK changes needed or is this the only one?

looks like no based on the epic

**Comment by oliviamiller** [comment]
> theres a change in app that depends on api but that should be a small one

okay I can wait then

**Comment by JohnN193** [comment]
kk looks like app is blocked on this change as well. so will wait to merge app until after this pr

**Comment by martha-johnston** [comment]
okay I'll merge this once api is ready then

**Pull request merged**
